bin/deploy

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+export ETH_GAS=2000000
+export GAS_PRICE=1gwei
+
+export DAPP_BUILD_OPTIMIZE_RUNS=1
```

Comment:
For a contract like this I would optimize towards many runs, so I'd set this to `200`